### ISSUE TITLE
share + export

### ISSUE THREAD

**Issue by ChrizKu** [nrenner/brouter-web#299]
add share features
- link (optional: custom link)
- <> embed

combine them with existing export features in the menu,
rename it: _share_ instead of _export_ with a suitable icon

link should include parameters #265
alternative to save trips #130

**Comment by nrenner** [comment]
I also thought about having a link with the proposed/edited name in the export dialog, which would help creating a bookmark with the name already set.

> rename it: share instead of export with a suitable icon

Not sure about renaming though, just as "share" is more appropriate for links, it's less so for exporting/downloading/saving the track.

**Comment by EssBee59** [comment]
Is the use of "Custom profile" compatible with this feature?
If not (as I suppose), the feature should be deactivated in that case to avoid errors...

**Comment by nrenner** [comment]
Yes, until #26 is implemented, sharing a link while a custom profile is active should probably at least show a warning.

**Comment by EssBee59** [comment]
Hello,

Sorry, possibly I did not understand wich link exactly should be used for sharing a tour..

I retested today using a custom_profile:
My route:
http://brouter.de/brouter-web/#map=12/49.8632/9.1123/osm-mapnik-german_style&lonlats=8.951797,49.968889;9.138912,49.832226

Now, I unselect the parameter "consider_elevation", the route changes but the URL do not change...
http://brouter.de/brouter-web/#map=12/49.8632/9.1123/osm-mapnik-german_style&lonlats=8.951797,49.968889;9.138912,49.832226 

(in my understanding, only the link to download the gpx changed as it contains the custom_profile/temporary  name)

For that reason, I agree with nrenner not to recommand this method to share routes with friends...

**Comment by EssBee59** [comment]
My error, it seems that the solution proposed was based on the "download link", not the creation link...
Anyway, I think sending the gpx file itself is a more secured way for transfering a tour!

**Comment by telemaxx** [comment]
would be cool if we could give a tour a name, which becomes part of the url. this name should than used for gpx export and saving a bookmark. 
https://www.mtb-news.de/forum/t/brouter-web-fragen-antworten-hilfe-profile-tipps-etc.917910/post-17004934
